app/src/main/java/com/pplnostrateam/arisyaag/insantani/Order.java

```diff
@@ -86,7 +115,108 @@ public void onClick(View v) {
                 Intent intent = new Intent(v.getContext(), Ordermade.class);
                 startActivityForResult(intent, 0);
             }
-        });
+        });*/
+    }
+
+    private void attemptCreateOrder() {
+        if (mOrderTask != null) {
+            return;
+        }
+
+        //showProgress(true);
+        mOrderTask = new OrderTask(1, "far away", "no note");
+        mOrderTask.execute((Void) null);
+    }
+
+
+    public class OrderTask extends AsyncTask<Void, Void, Boolean> {
+
+        private final int user;
+        private final String location;
+        private final String note;
+
+        OrderTask(int user, String location, String note) {
+            this.user = user;
+            this.location = location;
+            this.note = note;
+        }
+
+        @Override
+        protected Boolean doInBackground(Void... params) {
+            // attempt authentication against a network service.
+
+            try {
+                // Simulate network access.
+                Thread.sleep(2000);
+            } catch (InterruptedException e) {
+                return false;
+            }
+
+
+            try {
+                callServerResponse(user, location, note);
+            } catch (Exception e) {
+                return false;
+            }
+
+            return true;
+        }
+
+        @Override
+        protected void onPostExecute(final Boolean success) {
+            mOrderTask = null;
+            //showProgress(false);
+
+            if (success) {
+                startActivity(new Intent(Order.this, Ordermade.class));
+
+                //Toast.makeText(getApplicationContext(), "Order has been successfully added.", Toast.LENGTH_LONG).show();
+
+                finish();
+            } else {
+                // Toast.makeText(getApplicationContext(), "Register order failed...", Toast.LENGTH_LONG).show();
+
+            }
+        }
+
+        @Override
+        protected void onCancelled() {
+            mOrderTask = null;
+            //showProgress(false);
+        }
+    }
+
+    public void callServerResponse(int user, String location, String note) throws JSONException {
+
+        String json = String.format("{\"user\":%d,\"vegetable\":1,\"longitude\":108.100,\"latitude\":-6.100,\"location\":\"%s\",\"note\":\"%s\",\"quantity\":2,\"price\":4000}", user, location, note);
```

Comment:
Converting json to map of string may can be done by using Gson library, example
Gson gson = new Gson(); 
String json = "{\"k1\":\"v1\",\"k2\":\"v2\"}";
Map<String,Object> map = new HashMap<String,Object>();
map = (Map<String,Object>) gson.fromJson(json, map.getClass());